spec/support/features/webauthn_helper.rb

```diff
@@ -137,4 +188,8 @@ def signature
       B3/YJKhlr2AA5uw59+aFzk
     HEREDOC
   end
+
+  def platform_authenticator?
```

Comment:
I think I have a soft preference for the current approach, for convenience and since the helpers already make many other assumptions of "just working" for both platform authenticators and security keys. Open to revisiting though.